components/groups/read/group-content-tabs/tab.tsx

```diff
@@ -0,0 +1,51 @@
+import React, { Fragment, memo, useMemo } from 'react'
+
+import ContentDetailView from '@components/content/read-content/content-detail-view/content-detail-view'
+import { ContentTypeDefinition } from '@lib/types/contentTypeDefinition'
+import LoadingItems from '@components/generic/loading/loading-items'
+import EmptyList from '@components/generic/empty-list'
+import useSWR from 'swr'
+import API from '@lib/api/api-endpoints'
+import fetch from '@lib/fetcher'
+
+interface Props {
+  id: string
+  groupType: string
+  contentType: string | number
+  currentContentType: ContentTypeDefinition
+}
+
+function Tab({ id, contentType, groupType, currentContentType }: Props) {
+  const { data } = useSWR(
+    `${API.content[contentType]}?groupId=${id}&groupType=${groupType}`,
```

Comment:
Here instead of using the manual fetching and etc, can you use the contentListView? Content list view already has infinite pagination and things like that. 

ContentListView
            infiniteScroll={false}
            type={record}
            query={`author=${data?.id || null}`}
          />

The same like in profile/userId